cypress/views/tab.ts

```diff
@@ -29,51 +34,77 @@ export const tab = {
   navigateToConsole: () => {
     navigateToTab(tabs.Console);
   },
+  // vm configuration sub tabs
+  navigateToDetails: () => {
+    navigateToTab(tabs.Configuration);
+    cy.byButtonText('Details').click();
```

Comment:
I believe it would be better to have cypress find elements by some other property like `data-test` or some other property if missing rather than text found on screen as it has more chance to be changed in the future, WDYT?

Reply:
Again, the `data-test` or `data-test-id` is missing on these elements.

Reply:
see comment above :)

Reply:
I opened the issue https://issues.redhat.com/browse/CNV-52246 and assigned it to you.